src/ripple/app/tests/MultiSign.test.cpp

```diff
@@ -394,6 +412,209 @@ class MultiSign_test : public beast::unit_test::suite
         BEAST_EXPECT(env.seq(alice) == aliceSeq + 1);
     }
 
+    void test_regularSignersUsingSubmitMulti()
+    {
+        using namespace jtx;
+        Env env(*this, features(featureMultiSign));
+        Account const alice {"alice", KeyType::secp256k1};
+        Account const becky {"becky", KeyType::ed25519};
+        Account const cheri {"cheri", KeyType::secp256k1};
+        env.fund(XRP(1000), alice, becky, cheri);
+        env.close();
+
+        // Attach signers to alice.
+        env(signers(alice, 2, {{becky, 1}, {cheri, 1}}), sig (alice));
+
+        // Give everyone regular keys.
+        Account const beck {"beck", KeyType::secp256k1};
+        env(regkey (becky, beck));
+        Account const cher {"cher", KeyType::ed25519};
+        env(regkey (cheri, cher));
+        env.close();
+
+        // Disable cheri's master key to mix things up.
+        env(fset (cheri, asfDisableMaster), sig(cheri));
+        env.close();
+
+        auto const baseFee = env.current()->fees().base;
+        std::uint32_t aliceSeq;
+
+        // these represent oft-repeated setup for input json below
+        auto setup_tx = [&]() -> Json::Value {
```

Comment:
Oh yes, missed the env and basefee.
`[&]` not a biggie but better to avoid any capture if possible; it's more pure functional and all local so lookup is minimised.